package-lock.json

```diff
@@ -0,0 +1,2592 @@
+{
```

Comment:
We already have yarn.lock file and having 2 lockfiles is considered a bad practice.
Please remove it and use yarn to install dependencies.

Reply:
Fixed